src/icons/general/IconSharedLink2.js

```diff
@@ -0,0 +1,67 @@
+// @flow
```

Comment:
This is a new icon because I couldn't find a way to retrofit our existing one and preserve backward compatibility unfortunately.